src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -82,7 +82,7 @@ abstract class Kernel implements KernelInterface, RebootableInterface, Terminabl
     public const RELEASE_VERSION = 1;
     public const EXTRA_VERSION = 'DEV';
 
-    public const END_OF_MAINTENANCE = '05/2021';
+    public const END_OF_MAINTENANCE = '11/2021';
```

Comment:
```suggestion
    public const END_OF_MAINTENANCE = '01/2022';
```

Reply:
why ?

Reply:
Because https://symfony.com/releases/5.3

Reply:
@derrabus Maybe you're thinking about EOL date, which is - I think - already okay?

Reply:
for non-LTS release, the EOL and EOM dates are the same now

Reply:
My bad, totally forgot that. Fixed :)

Reply:
EOL and EOM is the same for x.0, x.1, x.2 and x.3. 

I agree with this change.